plugins/modules/site_workflow_manager.py

```diff
@@ -413,23 +497,25 @@ def get_current_site(self, site):
         """
 
         site_info = {}
-
-        location = get_dict_result(site[0].get("additionalInfo"), 'nameSpace', "Location")
+        location = get_dict_result(site[0].get(
+            "additionalInfo"), 'nameSpace', "Location")
         typeinfo = location.get("attributes").get("type")
 
         if typeinfo == "area":
             site_info = dict(
                 area=dict(
                     name=site[0].get("name"),
-                    parentName=site[0].get("siteNameHierarchy").split("/" + site[0].get("name"))[0]
+                    parentName=site[0].get("siteNameHierarchy").split(
```

Comment:
Why you split the existing code into multiple lines ?

Reply:
In sanity check its showing length is too long